articles/libraries/standard/data-structures.md

```diff
@@ -36,7 +36,7 @@ This can be used to change how a quantum register is represented when calling op
 ```qsharp
 let leRegister = LittleEndian(register);
 // QFT expects a BigEndian, so we can reverse before calling.
-QFT(BigEndian(Reverse(leRegister)));
+QFT(BigEndian(Reverse(leRegister!)));
```

Comment:
This takes Microsoft.Quantum.Arithmetic being open (as does using `LittleEndian` at all).
```suggestion
QFT(BigEndian(LittleEndianAsBigEndian(leRegister)));
```